### PR TITLE
SubscriberService kestrel + Core boostrapping plus mock GET endpoint

### PR DESCRIPTION
**Pull request by digimaun**

### CODE REVIEW

File: src/ServiceFabricPubSub/SubscriberService/PackageRoot/Config/Settings.xml [hunk 1]
```diff
@@ -1,16 +1,9 @@
...
+  <!--
+  <Section Name="MyConfigSection">
+    <Parameter Name="MyParameter" Value="Value1" />
   </Section>
-  <!-- This is used for securing StateManager's replication traffic. -->
-  <Section Name="ReplicatorSecurityConfig" />
-
-  <!-- Add your custom configuration sections and parameters here. -->
-  
-  <Section Name="SubscriberConfiguration">
```
Comment (PiDiBi) [inline src/ServiceFabricPubSub/SubscriberService/PackageRoot/Config/Settings.xml]: why did you delete it? it's the way, how to pass, Topic name to Subscriber via configuration file

File: src/ServiceFabricPubSub/SubscriberService/packages.config [hunk 2]
```diff
@@ -1,8 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (PiDiBi) [inline src/ServiceFabricPubSub/SubscriberService/packages.config:1]: why did you delete nuget references?

File: src/ServiceFabricPubSub/SubscriberService/PackageRoot/Config/Settings.xml [hunk 3]
```diff
@@ -1,16 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <Settings xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://schemas.microsoft.com/2011/01/fabric">
-  <!-- This is used by the StateManager's replicator. -->
-  <Section Name="ReplicatorConfig">
-    <Parameter Name="ReplicatorEndpoint" Value="ReplicatorEndpoint" />
```
Comment (PiDiBi) [inline src/ServiceFabricPubSub/SubscriberService/PackageRoot/Config/Settings.xml]: why did you delete this?

File: src/ServiceFabricPubSub/SubscriberService/PackageRoot/ServiceManifest.xml [hunk 4]
```diff
@@ -28,12 +29,8 @@
       <!-- This endpoint is used by the communication listener to obtain the port on which to 
            listen. Please note that if your service is partitioned, this port is shared with 
            replicas of different partitions that are placed in your code. -->
-      <Endpoint Name="ServiceEndpoint" />
-
-      <!-- This endpoint is used by the replicator for replicating the state of your service.
-           This endpoint is configured through a ReplicatorSettings config section in the Settings.xml
-           file under the ConfigPackage. -->
-      <Endpoint Name="ReplicatorEndpoint" />
```
Comment (PiDiBi) [inline src/ServiceFabricPubSub/SubscriberService/PackageRoot/ServiceManifest.xml]: why?

File: src/ServiceFabricPubSub/SubscriberService/packages.config (hunk 2 above)
Comment (digimaun) [reply src/ServiceFabricPubSub/SubscriberService/packages.config:1]: It's being refactored to a .NET core app, the project references are in the csproj.

File: src/ServiceFabricPubSub/SubscriberService/PackageRoot/Config/Settings.xml (hunk 3 above)
Comment (digimaun) [reply src/ServiceFabricPubSub/SubscriberService/PackageRoot/Config/Settings.xml]: It's part way towards a .NET core refactor, its work in progress - will port over.

File: src/ServiceFabricPubSub/SubscriberService/PackageRoot/ServiceManifest.xml (hunk 4 above)
Comment (digimaun) [reply src/ServiceFabricPubSub/SubscriberService/PackageRoot/ServiceManifest.xml]: It started as a .Net Core Web API project,in order to more quickly add Kestrel and a Controller. This configuration can quickly be added in the future commit.
